inc/class-rest-webhook-controller.php

```diff
@@ -113,7 +113,7 @@ public function handle_endpoint_response( WP_REST_Request $request ) {
 		}
 
 		// Pass to callback handler.
-		$result = call_user_func_array( $message_action['callback_or_url'], $payload );
+		$result = call_user_func_array( $message_action['callback_or_url'], [ $payload ] );
```

Comment:
We only needed to change anything where we were passing an associative array to call_user_func_array. This instance would probably be best handled as 
```suggestion
		$result = call_user_func_array( $message_action['callback_or_url'], array_values( $payload ) );
```